src/python/grapl_analyzerlib/tests/test_query_gen.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import pytest
+import hypothesis
+
+from grapl_analyzerlib.prelude import GraphClient
+from grapl_analyzerlib.nodes.lens import LensView, LensQuery
+
+
+@pytest.mark.integration_test
+class TestQueryGen(unittest.TestCase):
+    @hypothesis.settings(deadline=None)
+    @hypothesis.given(
+        lens_name=hypothesis.strategies.text(),
```

Comment:
Can we add a max_size to the text? Otherwise I've found that hypothesis can create absurdly large strings and it can actually slow things down needlessly. Even if it's like, 100, or 200, that can help.